dvc/render/image.py

```diff
@@ -30,15 +30,14 @@ def _write_image(
         img_path = os.path.join(
             static, f"{revision}_{filename.replace(os.sep, '_')}"
         )
-        rel_img_path = relpath(img_path, page_dir_path)
         with open(img_path, "wb") as fd:
             fd.write(image_data)
         return """
         <div>
             <p>{title}</p>
             <img src="{src}">
         </div>""".format(
-            title=revision, src=rel_img_path
+            title=revision, src=(relpath(img_path, page_dir_path))
```

Comment:
@jorgeorpinel I am not sure I understand. Do you mean to pass some HTML via `revision` CLI argument? What would this HTML be and what would be its purpose?

Reply:
Sorry for the delay @pared . 

> Do you mean to pass some HTML via revision CLI argument?

Yes.

> What would this HTML be and what would be its purpose?

A malicious `<script>`.

Reply:
Ok, so in the beginning, I did not understand the problem. Hmm, it seems that you are right here. 

On the other hand, I think that if one is able to use `dvc plots show`, he/she can edit the webpage in-place (since they have write permission anyway), so the modification can be done without help of `DVC`.